group_vars/all/support.yml

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
Of course it's possible, but it serves no purpose other than to confuse people or encourage them to edit the code tree. I would recommend it's removed.